tests/harness/cases/BUILD

```diff
@@ -17,6 +17,7 @@ proto_library(
         "bool.proto",
         "bytes.proto",
         "enums.proto",
+        "filename-with-dash.proto",
```

Comment:
This looks like the only meaningful change. Can you revert the extra noise from the formatting changes?

Reply:
Sure. Sorry, my editor did the auto formatting.

Reply:
Done.